packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -50,22 +53,31 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
   const { data: smartTitle, refetch } = useQuery({
     queryKey: key,
     queryFn: async () => {
+      let title = post?.title || post?.sharedPost?.title;
       // Enusre that we don't accidentally fetch the smart title for users outside of the feature flag
       if (!showPlusSubscription || !isLoggedIn) {
-        return post?.title || post?.sharedPost?.title;
+        return title;
       }
 
-      const data = await gqlClient.request<{
-        fetchSmartTitle: { title: string };
-      }>(POST_FETCH_SMART_TITLE_QUERY, {
-        id: post.sharedPost ? post.sharedPost.id : post?.id,
-      });
+      try {
+        const data = await gqlClient.request<{
+          fetchSmartTitle: { title: string };
+        }>(POST_FETCH_SMART_TITLE_QUERY, {
+          id: post.sharedPost ? post.sharedPost.id : post?.id,
```

Comment:
Not from this PR, but I think they should be the same.
```suggestion
          id: post.sharedPost?.id || post?.id,
```